crates/blockchain/error.rs

```diff
@@ -59,3 +62,42 @@ pub enum MempoolError {
     #[error("Mismatch between blob versioned hashes and blobs bundle content length")]
     BlobsBundleWrongLen,
 }
+
+#[derive(Debug)]
+pub enum ForkChoiceElement {
+    Head,
+    Safe,
+    Finalized,
+}
+
+impl Display for ForkChoiceElement {
```

Comment:
nit: using `derive` doesn't achieve the same thing?

Reply:
It doesn't but I found a way to leverage the derive of the `Debug` trait to not need to implement the `Display` one.